.Documentation/articles/developer/07-run-a-course.md

```diff
@@ -1,3 +1,93 @@
+# Create a Course
```

Comment:
I would make it a separate chapter and name it to "Create a Course Programmatically".

Reply:
By that I mean a separate file, 08-create-a-course-programmatically.md

Don't forget to update TOC and the rest of the file names.